src/openfermion/utils/_trotter_qasm_conversion_test.py

```diff
@@ -0,0 +1,183 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+
+'''
+Python's 'unittest' is run with either:
```

Comment:
unittest is compatible with pytest, but we prefer pytest.  Don't need to document this here as it is already in the Readme, but you can just using "pytest" in the directory you want to test, and it will find both types of test.

Reply:
well actually in the code we always use unittest (otherwise it would be a dependency) but we run the tests with pytest on Travis. Just remove this string.